src/implementations/twig/components/file/file-translation-item.html.twig

```diff
@@ -5,6 +5,7 @@
   - "title" (string) (default: '')
   - "meta" (string) (default: '')
   - "lang" (string) (default: '')
+  - "id" (string) (default: random)
```

Comment:
instead of providing a different id for each translation item, we could rely on the id coming from the file template, and add a suffix (something like '[id]-[language-code]' or [id]-1|2|3|...)
Similar to what has been done on the menu
https://github.com/ec-europa/europa-component-library/blob/v4-dev/src/implementations/twig/components/menu/menu.html.twig#L176

Reply:
ok, i've changed this id into {file_id}-translation-{key}